processor/queuedprocessor/queued_processor.go

```diff
@@ -133,6 +137,8 @@ func (sp *queuedSpanProcessor) ConsumeTraceData(ctx context.Context, td consumer
 	addedToQueue := sp.queue.Produce(item)
 	if !addedToQueue {
 		sp.onItemDropped(item, statsTags)
+	} else {
+		stats.RecordWithTags(context.Background(), statsTags, processor.StatBatchesDroppedCount.M(int64(0)))
```

Comment:
This will be executed every time an item is successfully added to the queue. Is it _necessary_ for this call to happen every time or we only need the 0 value to be published once at the startup?

I don't know what the cost of this call is, but it is in the hot path and I'd prefer not to do it unless it is required.

Nit: add a comment to explain why a value of 0 is published.

Reply:
Just once at startup, changing.